### DISCUSSION TITLE
How to upload Python code so everybody can see it newbie needs help

### DISCUSSION THREAD

**Discussion by hansemso** [community/community#44247] [New to GitHub] [answered]
I'm trying to make an algorithm using python.  New to programming, but know a ton of math, which i'm trying to use to make the algorithm.  How do I link my python file to github so that everybody can access it and work on it together?  I heard that's how it's done on github.  I created a file called "Newton's Method 2" as you can see.  But how do I put the python file in it?

**Comment by hansemso** [comment]
Oh wait.  Bad question.  I see it.  Thanks anyway.

**Comment by skadomsky** [comment]
Create a repo, upload your file, add a tag for your project, and that's all.
If you want to display them in a website, try integrating Markdown extensions.

**Comment by hansemso** [comment, marked as answer]
OK.  Thanks.  I'll look into it.
